src/BandcampDownloader/Properties/Resources.de.resx

```diff
@@ -0,0 +1,444 @@
+    <value>_Benachrichtigungen lautschalten</value>
+  </data>
+	<data name="checkBoxEnableApplicationSounds_ToolTip" xml:space="preserve">
+    <value>Wenn ausgewählt wird ein Benachrichtigungston nach erfolgreichem Download abgespielt.</value>
+  </data>
+	<data name="checkBoxModifyTags" xml:space="preserve">
+    <value>_ID3 tags editieren</value>
+  </data>
+	<data name="checkBoxModifyTags_ToolTip" xml:space="preserve">
+    <value>Wenn ausgewählt werden die ID3 tags nachträglich, wie unten beschrieben, modifiziert.
+
+Wenn nicht werden sie ignoriert und beim ursprünglichen Zustand beibehalten.
+Entspricht dem setzen von Nicht verändern bei jedem tag.</value>
+  </data>
+	<data name="checkBoxOneAlbumAtATime" xml:space="preserve">
+    <value>Ein album auf einmal _Herunterladen.</value>
+  </data>
+	<data name="checkBoxOneAlbumAtATime_ToolTip" xml:space="preserve">
+    <value>Wenn ausgewählt wird nur ein Album auf einmal heruntergeladen.
+Wenn nicht werden alle auf einmal heruntergeladen (Empfohlen wenn eine hohe bandbreite zur verfügung steht).</value>
+  </data>
+	<data name="checkBoxRetrieveFilesSize" xml:space="preserve">
+    <value>_Dateigrößen vorkalkulieren.</value>
+  </data>
+	<data name="checkBoxRetrieveFilesSize_ToolTip" xml:space="preserve">
+    <value>Wenn ausgewählt werden die Dateigrößen im vornherein abgerufen so dass eine genaue verbleibende Dauer berechnet werden kann.
+Das Herunterladen geht schneller wenn diese Option ausgeschaltet ist.</value>
+  </data>
+	<data name="checkBoxSaveCoverArtInFolder" xml:space="preserve">
+    <value>In _Ordner speichern</value>
+  </data>
+	<data name="checkBoxSaveCoverArtInFolder_ToolTip" xml:space="preserve">
+    <value>Wenn ausgewählt wird das Cover in den Album Ordner heruntergeladen.</value>
+  </data>
+	<data name="checkBoxSaveCoverArtInTags" xml:space="preserve">
+    <value>In _Tags speichern</value>
+  </data>
+	<data name="checkBoxSaveCoverArtInTags_ToolTip" xml:space="preserve">
+    <value>Wenn ausgewählt wird das Cover in den tracks gespeichert</value>
+  </data>
+	<data name="checkBoxVerboseLog" xml:space="preserve">
+    <value>_Ausführliches Log anzeigen</value>
+  </data>
+	<data name="checkBoxVerboseLog_ToolTip" xml:space="preserve">
+    <value>Wenn ausgewählt werden mehr Informationen ausgegeben</value>
+  </data>
+	<data name="folderBrowserDialogDescription" xml:space="preserve">
+    <value>Ordner auswählen um </value>
+  </data>
+	<data name="labelAlbumArtist" xml:space="preserve">
+    <value>Album Art_ist</value>
+  </data>
+	<data name="labelAlbumTitle" xml:space="preserve">
+    <value>Album _Name</value>
+  </data>
+	<data name="labelAllowedFileSizeDifference" xml:space="preserve">
+    <value>Erlaubte _dateigrößen unterschied in Prozent</value>
+  </data>
+	<data name="labelArtist" xml:space="preserve">
+    <value>_Artist</value>
+  </data>
+	<data name="labelButtonResetSettings" xml:space="preserve">
+    <value>Einstellungen _Zurücksetzen</value>
```

Comment:
Would "_Zurücksetzen" be enough for the context? That would allow me not to enlarge the button width:

![image](https://user-images.githubusercontent.com/3851902/55963731-aaea7f80-5c73-11e9-9b71-0ae1a5da6783.png)

Reply:
Yes, absolutely.